supervision/keypoint/skeletons.py

```diff
@@ -25,6 +25,30 @@ class Skeleton(Enum):
         (17, 15),
     ]
 
+    # Hardcoding here, but it also comes within model results
+    # Note: the classes in the model are 0-indexed, while the skeleton is 1-indexed
+    YOLO_NAS = [
```

Comment:
Let's drop this skeleton. The only difference between this and the COCO skeleton we already have is that it has 2 more edges that connect the head to the torso. That's why we should aim to annotate YOLO NAS result in the same way as YOLOv8.